doc/guides/modules.rst

```diff
@@ -487,6 +486,105 @@ variables. For example:
 
   include(${ZEPHYR_CURRENT_MODULE_DIR}/cmake/code.cmake)
 
+.. _modules_module_ext_root:
+
+Module ext root
```

Comment:
what does ext mean?. Can you spell this out? :)

Reply:
Please take a look at updated docs.